test/cbmc/proofs/DNS/DNSgetHostByName_a/DNSgetHostByName_a_harness.c

```diff
@@ -86,8 +89,67 @@ size_t prvCreateDNSMessage( uint8_t * pucUDPPayloadBuffer,
 
 void func( const char * pcHostName,
            void * pvSearchID,
-           uint32_t ulIPAddress )
+           struct freertos_addrinfo * pxAddressInfo )
 {
+    __CPROVER_assert( pcHostName != NULL,
+                      "Precondition: pcHostName != NULL" );
+    __CPROVER_assert( pvSearchID != NULL,
+                      "Precondition: pvSearchID != NULL" );
+}
```

Comment:
Any asserts on the `pxAddressInfo` pointer?

Reply:
fixed